deploy/helm/values.yaml

```diff
@@ -785,4 +785,29 @@ ebpfNetworkMonitoring:
 
 # Enable support for OpenShift environment
 openshift:
-  enabled: false
\ No newline at end of file
+  enabled: false
+
+operator:
+  enabled: false
+  manager:
+    image:
+      repository: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
+      tag: "0.107.0"
+    collectorImage:
+      # it is not yet supported to run collector instrumented by operator
+      # filling this to satisfy subchart requirement
+      repository: "solarwinds/swi-opentelemetry-collector"
+  admissionWebhooks:
+    certManager:
+      enabled: true
+      # ensure that Certificate and Issuer resource are created after CRD's of all dependencies
```

Comment:
Doesn't Helm create CRDs of a chart and all its dependencies before everything else?

Reply:
No, because the dependencies create CRDs through templates, not using Helm's support for CRDs.